forester/forester.toml

```diff
@@ -7,4 +7,8 @@ INDEXER_BATCH_SIZE=50
 INDEXER_MAX_CONCURRENT_BATCHES=50
 TRANSACTION_BATCH_SIZE=1
 TRANSACTION_MAX_CONCURRENT_BATCHES=50
-SLOT_UPDATE_INTERVAL_SECONDS=30
\ No newline at end of file
+SLOT_UPDATE_INTERVAL_SECONDS=30
+STATE_QUEUE_START_INDEX=0
+STATE_QUEUE_LENGTH=28807
+ADDRESS_QUEUE_START_INDEX=0
+ADDRESS_QUEUE_LENGTH=28807
```

Comment:
how easy is it to change these values?
Would it be easier to change if these are env variables?